portal_account/urls.py

```diff
@@ -22,4 +22,10 @@
         login_required(views.PortalProfileUpdate.as_view()),
         name="portal_profile_edit",
     ),
+    
+    path('sponsorship/', include('sponsorship.urls')),
```

Comment:
Instead adding this to `portal_account/urls.py` here, it should be added to `portal/urls.py`.
See example for adding the `/volunteer/` url: https://github.com/pyladies/pyladiescon-portal/blob/main/portal/urls.py#L28